packages/doltgres/content/reference/sql/version-control/dolt-system-tables.md

```diff
@@ -625,34 +678,30 @@ that a merge puts one or more tables in a state where constraints no longer hold
 merge base could be referenced via a foreign key constraint by an added row in the merged commit. Use
 `dolt.constraint_violations` to discover such violations.
 
-### Schema
+#### Schema
 
 ```sql
-+----------------+-----------------+------+-----+---------+-------+
-| Field          | Type            | Null | Key | Default | Extra |
-+----------------+-----------------+------+-----+---------+-------+
-| table          | text            | NO   | PRI |         |       |
-| num_violations | bigint unsigned | NO   |     |         |       |
-+----------------+-----------------+------+-----+---------+-------+
+     Field      |      Type       | Null | Key | Default | Extra
+----------------+-----------------+------+-----+---------+-------
+ table          | text            | NO   | PRI |         |
+ num_violations | bigint unsigned | NO   |     |         |
 ```
 
-# Rebasing Tables
+## Rebasing Tables
 
 ### `dolt.rebase`
 
 `dolt.rebase` (also usable as `dolt_rebase`) is only present while an interactive rebase is in progress, and only on the branch where the rebase is being executed. For example, when rebasing the `feature1` branch, the rebase will be executed on the `dolt_rebase_feature1` branch, and the `dolt.rebase` system table will exist on that branch while the rebase is in-progress. The `dolt.rebase` system table starts off with the default rebase plan, which is to `pick` all of the commits identified for the rebase. Users can adjust the rebase plan by updating the `dolt.rebase` table to change the rebase action, reword a commit message, or even add new rows with additional commits to be applied as part of the rebase. For more details about rebasing, see [the `dolt_rebase()` stored procedure](dolt-sql-procedures.md#dolt_rebase).
 
-### Schema
+#### Schema
 
-```text
-+----------------+---------------------------------------------------+
-| Field          | Type                                              |
-+----------------+---------------------------------------------------+
-| rebase_order   | DECIMAL(6,2)                                      |
-| action         | ENUM('pick', 'drop', 'reword', 'squash', 'fixup') |
-| commit_hash    | TEXT                                              |
-| commit_message | TEXT                                              |
-+----------------+---------------------------------------------------+
+```sql
+     Field      |                       Type                        | Null | Key | Default | Extra
+----------------+---------------------------------------------------+------+-----+---------+-------
+ rebase_order   | real                                              | NO   | PRI |         |
+ action         | enum('pick', 'drop', 'reword', 'squash', 'fixup') | NO   |     |         |
```

Comment:
We don't really have an enum type in Doltgres, not sure how this might work. Is it actually just a string?